source/ui/Image.js

```diff
@@ -188,6 +205,15 @@
 		/**
 		* @private
 		*/
+		placeholderChanged: function(){
+			if (this.placeholder) {
```

Comment:
In case of *img* tag, when there is no source set/exists, i.e. *!this.src* returns *true*, then only, I feel, unset/changed placeholder's value makes sense. If image source already exists, changing placeholder will remove the set source on img. 
Talking about *div* case, approach used considers multiple background image url's. If at later point of time, placeholder needs to be unset, then it will be done only when no source value exists.
Let me know your thoughts, if using some other approach we can achieve unsetting placeholder, no matter if src exists or not.